services/operatorsinfo/operatorsinfo_subgraph_integration_test.go

```diff
@@ -0,0 +1,100 @@
+package operatorsinfo
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"testing"
+)
+
+func TestStartAnvilChain(t *testing.T) {
+	pid, err := startAnvilChain()
+	if err != nil {
+		t.Fatal(err)
+	}
+	fmt.Println("Anvil chain started with PID", pid)
+
+	curr_path, err := os.Getwd()
+	changeDirectory(curr_path + "integration_test_deployment/lib/eigenlayer-middleware-offchain/subgraphs/BLSApkRegistry")
+
+	_, err = startGraph()
+	if err != nil {
+		t.Fatal(err)
+	}
+	_, err = startSubgraph()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// err = stopAnvilChain(pid)
+	// if err != nil {
+	// 	t.Fatal(err)
+	// }
+}
+
+func startAnvilChain() (int, error) {
+	return execCmd("anvil", []string{"--host", "0.0.0.0"}, []string{})
+}
+
+func startGraph() (int, error) {
+	return execCmd("docker", []string{"compose", "up"}, []string{})
```

Comment:
start anvil and graph using testcontainers like we do everywhere else. This way don't need to rely on other programs being installed on the user's computer, and also version will always be the same so won't have non-deterministic tests.